backend/game/cards/keep_cards/victory_point_manipulation_cards/test_urbavore.py

```diff
@@ -0,0 +1,42 @@
+from game.cards.keep_cards.victory_point_manipulation_cards.urbavore import Urbavore
+from game.values import locations
+
+
+def test_urbavore_costs_4_energy():
+    assert Urbavore().cost == 4
+
+
+def test_urbavore_gain_1_star_when_in_tokyo(player):
+    player.location = locations.Locations.TOKYO
+    Urbavore().special_effect(player, None)
+    assert player.victory_points == 1
+
+
+def test_urbavore_gains_no_stars_when_not_in_tokyo(player):
+    player.location = locations.Locations.OUTSIDE
+    Urbavore().special_effect(player, None)
+    assert player.victory_points == 0
+
+
+def test_urbavore_adding_to_attack_when_in_tokyo_and_attack_greater_than_one(player):
+    player.location = locations.Locations.TOKYO
+    attack = Urbavore().attack_dice_special_effect(player, 3)
```

Comment:
Being nitpicky, but I'd suggest making local variables to explain what the cards actually doing. Like `attack_rolled = 3` then `assert attack == attack_rolled + 1`